portingdb/check_drops.py

```diff
@@ -461,3 +533,18 @@ def check_drops(ctx, filelist, cache_sax, cache_rpms):
         with open(cache_dir / ('results-' + verdict + '.txt'), 'w') as f:
             for name in filtered:
                 print(name, file=f)
+
+    log('\nSource packages:')
+    stats_counter = collections.Counter(r['source_verdict'] for r in results.values())
+    for package, number in stats_counter.most_common():
+        log('{}: {}'.format(number, package))
+
+    for verdict in stats_counter:
+        if verdict == 'keep':
+            continue
+        filtered = {n: r for n, r in results.items() if r['source_verdict'] == verdict}
+        with open(cache_dir / ('results-' + verdict + '.json'), 'w') as f:
```

Comment:
Add `-srpms` here as well? I don't think we want to rewrite the earlier files.

Reply:
The verdicts differ and the data are collected by binary packages.

Reply:
the verdicts names I mean

Reply:
ok